app/(route)/playground/[uid]/_nav.tsx

```
@@ -0,0 +1,244 @@
+
+function ButtonAutoSave() {
+  const [autoSave, setAutoSave] = useState(false);
+
+  const handleAutoSave = () => {
+    setAutoSave(!autoSave);
+  };
+
+  return (
+    <div className="flex flex-row items-center gap-[10px]">
+      <div className="text-[16px] font-bold text-[#F5F5F5]">AUTOSAVE</div>
+      <div onClick={handleAutoSave}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)